src/models/addon.js

```diff
@@ -289,6 +285,24 @@ function parseAddonOptions (options) {
   }, {});
 }
 
+async function getPlan (planName, provider) {
+  // if no plan specified, pick the cheapest one
+  if (planName == null || planName === '') {
+    if (provider.plans.length === 0) {
+      throw new Error('no plans available');
+    }
+
+    return provider.plans.sort((p1, p2) => p1.price - p2.price)[0];
+  }
+
+  const plan = provider.plans.find((p) => p.slug.toLowerCase() === planName.toLowerCase());
+  if (plan == null) {
+    const availablePlans = provider.plans.map((p) => p.slug);
+    throw new Error(`invalid plan name. Available plans: ${availablePlans.join(', ')}`);
```

Comment:
nitpick: we don't have copy guidelines but I would have used an upper case letter at the beginning and a coma before the "available plans:".

Reply:
This was not the subject of this PR, but I can do that.

Reply:
Ah you're right, sorry. We'll discuss it in the meeting :+1: